test/common_utils.py

```diff
@@ -284,8 +289,29 @@ def __init__(
         mae=False,
         **other_parameters,
     ):
-        if all(isinstance(input, PIL.Image.Image) for input in [actual, expected]):
-            actual, expected = (to_image(input) for input in [actual, expected])
+        # Convert PIL images to tv_tensors.Image (regardless of what the other is)
+        if isinstance(actual, PIL.Image.Image):
+            actual = to_image(actual)
+        if isinstance(expected, PIL.Image.Image):
+            expected = to_image(expected)
+
+        # Convert CV-CUDA tensors to torch.Tensor (regardless of what the other is)
+        try:
+            import cvcuda
+            from torchvision.transforms.v2.functional import cvcuda_to_tensor
+
+            if isinstance(actual, cvcuda.Tensor):
+                actual = cvcuda_to_tensor(actual)
+                # Remove batch dimension if it's 1 for easier comparison
+                if actual.shape[0] == 1:
+                    actual = actual[0]
```

Comment:
This seems unnecessary, we should be able to compare tensors where the batch dim is 1. Try to remove it, if it doesn't work for any reason let me know.

Reply:
Are we able to split the logic to drop batch and move to cpu into a helper function? We do the same thing in the test itself, and I think it would improve clarity to have an explicit helper. 

[Line in reference](https://github.com/pytorch/vision/pull/9277/commits/1e5e9ede063a415b0b48084d02e68191dd3ac25e#diff-9c2dde92db86c123fee225e39b7c1ef96e08a3e79a9dcc9a2d68b21ed51a81d0L1313)